src/status_im/transport/message/core.cljs

```diff
@@ -43,11 +43,12 @@
         ^js pin-messages               (.-pinMessages response-js)
         ^js removed-messages           (.-removedMessages response-js)
         sync-handler                   (when-not process-async process-response)]
-
     (cond
 
       (seq chats)
       (do
+        (when (seq contacts)
```

Comment:
could you please add comment here

Reply:
@flexsurfer sorry for delay, comment is added